building/components/homeworld-admin-tools/src/virt.py

```diff
@@ -333,9 +333,9 @@ def boot_install_and_admit(self):
 
     def boot_install(self, bootstrap_token):
         self.create_disk()
-        # TODO: do something better than a two-second delay to detect "boot:" prompt
+        # TODO: do something better than a ten-second delay to detect "boot:" prompt
         bootline = ("install netcfg/get_ipaddress=%s homeworld/asktoken=%s\n" % (self.node.ip, bootstrap_token)).encode()
-        if self.boot_with_io("install", text=bootline, delay=2.0).wait():
+        if self.boot_with_io("install", text=bootline, delay=10.0).wait():
```

Comment:
Just for future reference, I suspect an easier fix than detecting the boot prompt is to prefix the bootline with a single space, because it'll only read in one character before reaching the boot prompt. But that's irrelevant for this changeset.